# HG changeset patch
# User lo sppedflyer@gmail.com
# Date 1476961242 -7200
#      Thu Oct 20 13:00:42 2016 +0200
# Branch CMake
# Node ID 73fc730619a6fa4b2047c2321e84b1bea53c185b
# Parent  bc079a6aebea7b38a0042f4b342c914b3c669a94
patch math_undefs.h

diff -r bc079a6aebea -r 73fc730619a6 include/Inventor/C/base/math-undefs.h
--- a/include/Inventor/C/base/math-undefs.h	Thu Oct 20 12:43:02 2016 +0200
+++ b/include/Inventor/C/base/math-undefs.h	Thu Oct 20 13:00:42 2016 +0200
@@ -56,24 +56,26 @@
 
 #ifndef __clang__
 
-#undef cosf
-#define cosf(x) NO_SINGLEPREC /* whatever that'll give us a compile error... */
-#undef sinf
-#define sinf(x) NO_SINGLEPREC
-#undef tanf
-#define tanf(x) NO_SINGLEPREC
-#undef powf
-#define powf(x) NO_SINGLEPREC
-#undef sqrtf
-#define sqrtf(x) NO_SINGLEPREC
-#undef acosf
-#define acosf(x) NO_SINGLEPREC
-#undef asinf
-#define asinf(x) NO_SINGLEPREC
-#undef atanf
-#define atanf(x) NO_SINGLEPREC
-#undef atan2f
-#define atan2f(x) NO_SINGLEPREC
+# if 0
+    #undef cosf
+    #define cosf(x) NO_SINGLEPREC /* whatever that'll give us a compile error... */
+    #undef sinf
+    #define sinf(x) NO_SINGLEPREC
+    #undef tanf
+    #define tanf(x) NO_SINGLEPREC
+    #undef powf
+    #define powf(x) NO_SINGLEPREC
+    #undef sqrtf
+    #define sqrtf(x) NO_SINGLEPREC
+    #undef acosf
+    #define acosf(x) NO_SINGLEPREC
+    #undef asinf
+    #define asinf(x) NO_SINGLEPREC
+    #undef atanf
+    #define atanf(x) NO_SINGLEPREC
+    #undef atan2f
+    #define atan2f(x) NO_SINGLEPREC
+#endif
 
 #endif // !__clang__
 
